.github/workflows/continuous-integration.yml

```diff
@@ -50,3 +53,11 @@ jobs:
 
       - name: Test with pytest
         run: pytest --cov=dfxml --cov-report=xml .
```

Comment:
Am I understanding this right: The coverage reported on codecov only pertains to what is reported by `pytest`, from this command?

Reply:
Correct. When pytest runs with the --cov= option, it creates a coverage report, which basically outputs to a file every line of every python file that is executed. So the coverage command literally shows you what was covered. You can also add things to the coverage configuration file to exclude things from coverage based on patterns and what not, to make you look better (or so you don't look bad for things that you can't possibly cover).

Reply:
Understood, thanks!